edualgo/backtracking_algorithms/n_queen.py

```diff
@@ -0,0 +1,187 @@
+# N-Queens Problem via Backtracking
+
+from __init__ import print_msg_box
+from time import time as ti
+n=0                                             #Size of the Board
```

Comment:
There is no need to take input, those will be passed by the user manualy as a parameter of the function.